runtime/executor/src/main/java/edu/snu/nemo/runtime/executor/task/TaskExecutor.java

```diff
@@ -417,18 +446,40 @@ private boolean handleDataFetchers(final List<DataFetcher> fetchers) {
         .collect(Collectors.toList());
   }
 
-  private List<OutputWriter> getChildrenTaskWriters(final int taskIndex,
-                                                    final IRVertex irVertex,
-                                                    final List<StageEdge> outEdgesToChildrenTasks,
-                                                    final DataTransferFactory dataTransferFactory) {
+  private List<OutputWriter> getMainChildrenTaskWriters(final int taskIndex,
```

Comment:
Please describe the difference of this method and `getAdditionalChilderenTaskWriters` as method-level comments.